orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
`Domain.variables` and `Domain.attributes + Domain.class_vars` used to be `std::vector<PVariable>` and you can't add STL vectors on the fly whenever you needed to traverse over attributes and class(es). Both were also kept in Orange 3 for convenience. The idea is to (consistently) use the former instead of adding. Also, `attributes`, `class_var` and `class_vars` should be read-only properties.

What is wrong with [`iter(domain)`](https://github.com/biolab/orange3/blob/master/Orange/data/domain.py#L226)?